### DISCUSSION TITLE
Generate RSA & ECDSA certificates at once

### DISCUSSION THREAD

**Discussion by marcstern** [acmesh-official/acme.sh#4346] [Q&A] [answered]
Hello. How do we generate both a  RSA and a ECDSA certificate for a site in a single shot? Thanks

**Comment by neilpang** [comment, marked as answer]
you need to use `--issue` command twice.

**Reply by BadgerBadgerAndFox** [reply]
Further to this is it possible to deploy both the RSA and ECC certificate as the default cert using the Synology deploy hook?

**Reply by BadgerBadgerAndFox** [reply]
To clarify, this would be something that would be handled by the Synology deploy hook, i have no issues in issuing and renewing both certs.

NGINEX supports dual certs with cert selection handled during negotiation.

Synology currently issues and binds dual ECC/RSA certificates for Quickconnect by default, so it appears that it is also supported by DSM.
